03_probability_intro/03_probability_intro.jl

```diff
@@ -171,22 +384,63 @@ This may sound a bit confusing at first, but if you look closely for some time a
+It is interesting to give a little more detail on the epistemological interpretation of this entire theorem. We start from some initial hypothesis and we assign some probability to it (the Prior). How exactly this has to be done is uncertain, in fact, there are ways to encode that you don't know anything about the validity of the hypothesis. But the important part is that, if you already knew something about it, you can include that in your priors. For example, if you are trying to estimate some parameter that you know is positive definite, then you can use priors that are defined only for positive values. From that starting point, then, you have a methodical way to update your beliefs by collecting data, and with this data, give rise to the Posterior probability, which hopefully will be more accurate than our Prior probability. 
 What is nice about the Bayesian framework is that we always account for the uncertainty of the world. We start with some probability and end with another probability, but uncertainty is always present. This is what real life is about.
 To understand the full power of Bayesian probability we have to extend the notion of probability to *probability density*. We will discuss this topic in the following section.
 "
 
 # ╔═╡ 2d9482ce-1252-11eb-0cc7-35ad9c288ef8
 md"
-# Probability density
+# Probability distributions
 So far, we have been talking of probabilities of particular events.
-Probability density, on the other hand, help us compute probabilities of various events.
-These are functions that connect each event in an 'event space' to some probability. What do we mean by 'event space'? For example, consider the density probability of heights of adult women, given approximately by a **Normal distribution**,
+Probability distributions, on the other hand, help us compute probabilities of various events.
+We can distinguish between discrete and continuous cases depending on the possible output of the experiment.
 "
 
-# ╔═╡ 351923ee-5436-11eb-2bf6-8d024a64e83e
+# ╔═╡ f36ea8bc-9623-11eb-24a2-155b112db75b
+md"
+## Discrete Case
+If the outputs of our experiment are discrete, then its distribution is called a probability mass function, where we assign a probability to each possible outcome.
+
+One of the most popular distributions is the Poisson distribution.
+Suppose I want to visualize the probability of receiving *x* spam mails on Mondays.
+"
+
+
+# ╔═╡ e369ca40-9624-11eb-3d71-7719b56e0559
 md"
+Here we represent the probability of receiving *x* spam mail in a day.
+The interpretation of this graph is pretty straightforward. 
+The probability of receiving 0 spam emails on a Monday is approximately 0.2, for 1 spam email is slightly higher than 0.3 and so on, we have the probability of each possible output.
+
+By replicating the code below in a Pluto notebook, you will be able to create sliders to play around with the values of μ and σ and see how the Poisson distribution changes.
```

Comment:
Poisson distribution has one unique parameter, lambda. Mu and sigma here are not correct